web/src/pages/apd/activities/cost-allocation/CostAllocateFFPQuarterly.test.js

```diff
@@ -388,6 +388,116 @@ describe('the cost allocation quarterly FFP component', () => {
+      {
+        initialState: {
+          apd: {
+            data: {
+              years: ['2022']
+            },
+            adminCheck: false
+          },
+          budget: {
+            activities: {
+              '6ea6b4a2': {
+                quarterlyFFP: {
+                  years: {
+                    2022: {
+                      1: {
+                        combined: {
+                          dollars: 827785,
+                          percent: 0
+                        },
+                        contractors: {
+                          dollars: 537029,
+                          percent: 0.29
+                        },
+                        inHouse: {
+                          dollars: 518510,
+                          percent: 0.28
+                        }
+                      },
+                      2: {
+                        combined: {
+                          dollars: 753712,
+                          percent: 0
+                        },
+                        contractors: {
+                          dollars: 388882,
+                          percent: 0.21
+                        },
+                        inHouse: {
+                          dollars: 407401,
+                          percent: 0.22
+                        }
+                      },
+                      3: {
+                        combined: {
+                          dollars: 753711,
+                          percent: 0
+                        },
+                        contractors: {
+                          dollars: 290756,
+                          percent: 0.25
+                        },
+                        inHouse: {
+                          dollars: 462955,
+                          percent: 0.25
+                        }
+                      },
+                      4: {
+                        combined: {
+                          dollars: 753711,
+                          percent: 0
+                        },
+                        contractors: {
+                          dollars: 290756,
+                          percent: 0.25
+                        },
+                        inHouse: {
+                          dollars: 462955,
+                          percent: 0.25
+                        }
+                      },
+                      subtotal: {
+                        combined: {
+                          dollars: 3014848,
+                          percent: 0
+                        },
+                        contractors: {
+                          dollars: 1163026,
+                          percent: 1
+                        },
+                        inHouse: {
+                          dollars: 1851822,
+                          percent: 1
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    );
+
+    expect(
```

Comment:
for these tests you should try setting the value to 29 and 28 because that's where the problem was happening

Reply:
I did on lines 415 and 419 :)